### PR TITLE
Add Like and Unlike Endpoints with Tests

### PR DESCRIPTION
**Pull request by davidjnevin**

### CODE REVIEW

File: tests/api/routes/test_proposals.py [hunk 1]
```diff
@@ -95,3 +98,59 @@ def test_create_proposal(client, reverse_url):
...
+        .with_liked_by([fanboy])
+        .build()
+    )
+    url = reverse_lazy("api-alpha:like_proposal", kwargs={"proposal_id": proposal.id})
+    client.login(username=fanboy.email, password=password)
+
+    response = client.delete(url, data={}, content_type="application/json")
+
+    assert_that(response.status_code, equal_to(HTTPStatus.NO_CONTENT))
+    liked_by = proposal.liked_by.all()
```
Comment (aalmiramolla) [inline tests/api/routes/test_proposals.py:155]: Add a proposal.refresh_from_db() to be confident 100%

File: meetupselector/proposals/services/proposal.py [hunk 2]
```diff
@@ -38,3 +39,19 @@ def create(
...
+        proposal_to_like.liked_by.add(user)
+    return True
+
+
+def unlike(proposal_id: UUID4, user_id: UUID4) -> bool:
+    user = get_object_or_404(User, pk=user_id)
+    proposal_to_like = get_object_or_404(Proposal, pk=proposal_id)
+    if user in proposal_to_like.liked_by.all():
+        proposal_to_like.liked_by.remove(user)
+    return True
```
Comment (aalmiramolla) [inline meetupselector/proposals/services/proposal.py:57]: Remove return and return typing

File: meetupselector/api/schemas/proposals.py [hunk 3]
```diff
@@ -1,7 +1,10 @@
-from ninja import ModelSchema
+from uuid import uuid4
+
+from ninja import ModelSchema, Schema
 from pydantic import UUID4
 
 from meetupselector.proposals.models import Proposal
+from meetupselector.proposals.services.proposal import like
 
 from .talks import TopicRetrieveSchema
```
Comment (aalmiramolla) [inline meetupselector/api/schemas/proposals.py:10]: Revert this file to previous state

**Review by aalmiramolla — CHANGES_REQUESTED** [review]
Also add a test:
Given a proposal without votes
When a not logged user likes the proposal
Then an 403 response should be received

File: meetupselector/api/schemas/proposals.py (hunk 3 above)
Comment (davidjnevin) [reply meetupselector/api/schemas/proposals.py:10]: OK

File: meetupselector/proposals/services/proposal.py (hunk 2 above)
Comment (davidjnevin) [reply meetupselector/proposals/services/proposal.py:57]: OK

File: tests/api/routes/test_proposals.py (hunk 1 above)
Comment (davidjnevin) [reply tests/api/routes/test_proposals.py:155]: OK

**Review by aalmiramolla — APPROVED** [review]
LGTM

**Pull request merged**